src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -205,11 +200,10 @@ public Pose2d getPose() {
 		return odometer.getEstimatedPosition();
 	}
     public void resetOdometry(Pose2d pose) {
-		zeroGyroscope(pose.getRotation().getDegrees());
-        odometer.resetPosition(pose.getRotation(), getModulePositions(), pose);
+        odometer.resetPosition(getGyroscopeRotation(), getModulePositions(), pose);
     }
     public void resetOdometry() {
-        odometer.resetPosition(getGyroscopeRotation(), getModulePositions(), DRIVE_ODOMETRY_ORIGIN);
+		resetOdometry(DRIVE_ODOMETRY_ORIGIN);
```

Comment:
I don't quite understand how the odometry reset works... Is there a need for this to be different between red and blue alliances?